vertx-web/pom.xml

```diff
@@ -46,6 +46,18 @@
       <optional>true</optional>
     </dependency>
 
+    <!-- Validation dependencies -->
```

Comment:
these dependencies should go with the oas module pom not the root pom

Reply:
They are only about json validation, that is included inside vertx-web module

Reply:
Yes I understand but they should be declared on the module Pom not the top level because otherwise users not using oas will get them included in their projects without needing it